docs/reference/analysis/tokenfilters/condition-tokenfilter.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[analysis-condition-tokenfilter]]
+=== Conditional Token Filter
+
+The conditional token filter takes a predicate script and a list of subfilters, and
+only applies the subfilters to the current token if it matches the predicate.
+
+[float]
+=== Options
+[horizontal]
+filters:: a list of token filters to apply to the current token if the predicate
```

Comment:
We should be explicit that these filters are chained. Maybe call it `filter` (no plural) for consistency with analyzers?